### PR TITLE
Release 1

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following

- [x] I have used [Covid-19 API](https://www.postman.com/postman/workspace/covid-19)
- [x] Build Home and Details Pages.
- [x] Style Home and Details Pages.

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const { continent } = useSelector((state) => state.continentReducer);
+
+  return (
+    <nav className={classes.nav}>
+      {location.pathname === '/' && (
+        <>
+          <p>
+            <b>The World</b>
+          </p>
+          <p className={classes.shrink}>Continents COVID Cases</p>
```
Comment (er-contreras) [inline src/components/Navbar/Navbar.js:24]: - [x] When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name. Please don't forget to add a search bar. 😉

File: src/components/Country/Country.module.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.countryContainer:nth-child(odd) { background-color: #5582e1; }
+.countryContainer:nth-child(even) { background-color: #4169af; }
+
+.img {
+  width: 40px;
+  height: 50%;
+  border: 1px solid white;
+}
```
Comment (er-contreras) [inline src/components/Country/Country.module.css:16]: - [x] The Images of every country can be improved as it seems a little distorted.
Screen Shot 2022-02-02 at 19 26 46

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @keroloussamy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️

- [x] Your app is overflowing a little bit it would be great just to see the blue background.

Screen Shot 2022-02-02 at 19 24 50


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (keroloussamy) [reply src/components/Navbar/Navbar.js:24]: Done

Comment (keroloussamy) [reply src/components/Navbar/Navbar.js:24]: Done

File: src/components/Country/Country.module.css (hunk 2 above)
Comment (keroloussamy) [reply src/components/Country/Country.module.css:16]: Done

**Comment by omar-labana** [comment]
Hi @keroloussamy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**